src/pages/error/ErrorPage.tsx

```
@@ -0,0 +1,26 @@
+import Error from "@/assets/error/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)